nlp/README.md

```diff
@@ -20,3 +20,7 @@ conda init
 conda activate birdy
 python <python_file.py>
 ```
+
+# Model flair
+The "ner-large" flair model is included in the image build. This means that the models will not be updated to the latest.
```

Comment:
Je suggèrerais juste une reformulation ici avec un peu + de détails, pour que ce soit un peu + clair par rapport au contexte de l'explication (tu peux modifier ma suggestion au besoin):

The "ner-large" flair model is included in the image build to prevent writing permission problems occuring with the SequenceTagger.load() function, which would download the latest model online and try to save it on a read-only directory on the Docker container. This means that the models will not be updated to the latest version until the next image build.
If the latest version is needed, the image will have to be rebuilt with --no-cache.

Reply:
Tout à fait d'accord avec ta proposition !